src/jobbergate_cli_ops.py

```diff
@@ -64,20 +63,24 @@ def install(self):
         subprocess.call(["./src/templates/install_pyyaml.sh"])
 
         # Install package from private pypi
+        package_version = self._charm.model.config.get("version")
+        target_package = self._PACKAGE_NAME
+        if package_version:
+            target_package += f"=={self._charm.model.config['version']}"
         pip_install_cmd = [
             self._PIP_CMD,
             "install",
-            "-f",
+            "--index-url",
             self._derived_pypi_url(),
-            self._PACKAGE_NAME,
+            target_package,
         ]
         out = subprocess.check_output(pip_install_cmd).decode().strip()
         if "Successfully installed" not in out:
             logger.error(
-                f"Trouble installing {self._PACKAGE_NAME}, please debug"
+                f"Trouble installing {target_package}, please debug"
```

Comment:
```suggestion
                f"Error installing {target_package}"
```
Log messages should be short

Reply:
Addressed in commit b81d10be6ecf8856781e012b68862765801548a5